scripts/component_readme/generate_readme.py

```diff
@@ -21,14 +20,17 @@ def generate_readme(component_spec: ComponentSpec, *, component_dir: Path) -> st
     return template.render(
         id=component_dir.name,
         name=component_spec.name,
-        component_folder_name=component_spec.component_folder_name,
+        component_folder_name=component_spec.name,
```

Comment:
Can we remove the `component_folder_name` argument and just use the `id` everywhere in its place instead?